FluentTerminal.App/Services/Implementation/DefaultValueProvider.cs

```diff
@@ -406,11 +406,11 @@ public IEnumerable<TerminalTheme> GetPreInstalledThemes()
                 PreInstalled = true,
                 Colors = new TerminalColors
                 {
-                    Foregroun = "#EEEEEE",
+                    Foreground = "#EEEEEE",
                     Background = "#300A24",
                     Cursor = "#CFF5DB",
                     CursorAccent = "#000000",
-                    Selection = "rgba(207, 245, 219, 0.99609375)",
+                    Selection = "4DABEDC1",
```

Comment:
xterm.js does not support #aarrggbb, try rgba(207, 245, 219, 0.33) in this case

Reply:
Hmm, but just above, the homebrew theme uses it as well 🤔

Reply:
I thought there was a problem, but maybe it was in a previous version of xterm.js, sorry😅

Reply:
you just merged while i pushed a commit 😓. Because I also noticed that i forgot the `#`

Reply:
No worries, I will add it later today